spec/models/school_spec.rb

```diff
@@ -7,24 +7,30 @@
   it { should have_many(:school_group_memberships) }
   it { should have_many(:school_groups) }
 
-  describe '#has_religious_character?' do
-    before do
-      subject.gias_data = {}.to_json
-    end
+  describe '#religious_character' do
+    let(:religious_character) { 'Roman Catholic' }
+    let(:gias_data) { { 'ReligiousCharacter (name)' => religious_character } }
+
+    before { allow(subject).to receive(:gias_data).and_return gias_data }
```

Comment:
I think this needs some refactoring. I don't understand why a normal field like `gias_data` is managed this way. we should just have something like

```
    subject { build(:school, gias_data: gias_data) }

    let(gias_data) { {...} }
```